tests/integration/test_swapaxes.py

```diff
@@ -23,34 +24,130 @@
 
 def test_small():
     a_num = num.array(a)
+    b = a.swapaxes(0, 1)
     b_num = a_num.swapaxes(0, 1)
+    assert np.array_equal(b, b_num)
 
-    assert num.array_equal(a_num.sum(axis=0), b_num.sum(axis=1))
+    c = np.swapaxes(a, 0, 1)
+    c_num = num.swapaxes(a_num, 0, 1)
+    assert np.array_equal(c, c_num)
```

Comment:
All of these tests seem like they should be split up: a test for the method, and a test for the module function.

Reply:
I'm not sure if it's necessary to test both num.method(array, args) and array.method(args) for every test case.
The definition of num.method is like
`def method(arrary, args):`
`    return array.method(args)`
So it may be enough to test num.method.

Reply:
There's no guarantee that will always be the case in the future, and the purpose of having the tests now is so that if there are ever refactorings or changes in the future, we will have the tests to ensure those change don't break anything.

Reply:
Fixed.